README.md

```diff
@@ -174,6 +174,9 @@ The general workflow is the following:
     }
     ```
 
+## Tips
+When working with NEAR smart contracts you will need to handle many large numbers (the minimum allowed amount of NEAR is 1/10<super>24</super> of one NEAR token – put another way, one NEAR token is 1,000,000,000,000,000,000,000,000 yoctoNEAR). To simplify handling such large numbers, we recommend [decimate](https://docs.rs/decimate/).
+
```

Comment:
It's not only for large numbers but also for decimal numbers (eg: `1.25`) which are not represented as `float`.

Let's specify this as well.

Reply:
Good call; let me know if you like the updated version